### PR TITLE
Fix some errors and typos in the change log

### PR DESCRIPTION
**Pull request by hbushouse**
This PR simply fixes some errors and formatting/typo/grammar issues in the change log, which I noticed while compiling release notes for Cal 1.6.2 (DMS 8.1).

**Checklist for maintainers**
- [ ] added entry in `CHANGES.rst` within the relevant release section
- [ ] updated or added relevant tests
- [ ] updated relevant documentation
- [ ] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -4,78 +4,87 @@
...
+  from the Poppy package, which was recently updated to use a different sign convention.
+  [#6967]
 
 assign_wcs
 ----------
 
 - Added convenience function ``update_fits_wcsinfo()`` to ``assign_wcs.util``
   module to allow easy updating of FITS WCS stored in ``datamodel.meta.wcsinfo``
-  from data model's GWCS. [#6935]
+  from datamodel's GWCS. [#6935]
```
Comment (zacharyburnett) [inline CHANGES.rst:22]:
```suggestion
  from ``datamodel``'s GWCS. [#6935]
```

File: CHANGES.rst [hunk 2]
```diff
@@ -4,78 +4,87 @@
...
 ----------
 
 - Remove trailing dash from IFU cube filenames built from all subchannels.
-  Sort subchannels present by inverse alphabetical order to ensure
+  Also sort subchannels present by inverse alphabetical order to ensure
   consistent filename creation across processing runs. [#6959]
 
-- Update the wcs value for naxis3 when wavelength planes are removed from the
-  IFUCube when there is no valid data. [#6976]
+- Update the WCS naxis3 value when wavelength planes are removed from the
```
Comment (zacharyburnett) [inline CHANGES.rst:31]:
```suggestion
- Update the WCS ``naxis3`` value when wavelength planes are removed from the
```

File: CHANGES.rst [hunk 3]
```diff
@@ -4,78 +4,87 @@
...
+  IFUCube due to no valid data. [#6976]
 
 datamodels
 ----------
 
-- Updated keyword titles in data model schemas to match those in keyword
+- Updated keyword comments/titles in datamodel schemas to match those in keyword
```
Comment (zacharyburnett) [inline CHANGES.rst:37]:
```suggestion
- Updated keyword comments/titles in ``datamodel`` schemas to match those in keyword
```

File: CHANGES.rst [hunk 4]
```diff
@@ -4,78 +4,87 @@
...
   dictionary. [#6941]
 
-- Add P_SUBARR keyword to the `DarkModel` schema. [#6951]
+- Add the P_SUBARR keyword to the `DarkModel` schema. [#6951]
```
Comment (zacharyburnett) [inline CHANGES.rst:40]:
```suggestion
- Add the ``P_SUBARR`` keyword to the ``DarkModel`` schema. [#6951]
```

File: CHANGES.rst [hunk 5]
```diff
@@ -4,78 +4,87 @@
...
 
-- Added the P_READPA keyword to the `ReadnoiseModel` schema [#6973]
+- Add the P_READPA keyword to the `ReadnoiseModel` schema [#6973]
```
Comment (zacharyburnett) [inline CHANGES.rst:42]:
```suggestion
- Add the ``P_READPA`` keyword to the ``ReadnoiseModel`` schema [#6973]
```

File: CHANGES.rst [hunk 6]
```diff
@@ -4,78 +4,87 @@
...
 
 extract_1d
 ----------
 
-- Update int_times keywords in SOSS spectral output [#6930]
+- Update int_times keywords and copy the INT_TIMES table extension to SOSS
```
Comment (zacharyburnett) [inline CHANGES.rst:47]:
```suggestion
- Update ``int_times`` keywords and copy the ``INT_TIMES`` table extension to SOSS
```

File: CHANGES.rst [hunk 7]
```diff
@@ -4,78 +4,87 @@
...
+  spectral output (x1d) files [#6930]
 
 master_background
 -----------------
 
-- Fix MRS sigma-clipped background use in cases where EXTENDED keyword not
-  properly set. [#6960]
+- Fix the use of MRS sigma-clipped background in cases where the SRCTYPE
```
Comment (zacharyburnett) [inline CHANGES.rst:53]:
```suggestion
- Fix the use of MRS sigma-clipped background in cases where the ``SRCTYPE``
```

File: CHANGES.rst [hunk 8]
```diff
@@ -4,78 +4,87 @@
...
+- Fix a bug so that computed background values eare subtracted from the image
+  data when ``subtract=True``. [#6934]
 
 tweakreg
 --------
 
-- ``tweakreg`` step now updates FITS WCS stored in ``datamodel.meta.wcsinfo``
-  from data model's tweaked GWCS. [#6936, #6947, #6955]
+- The ``tweakreg`` step now updates FITS WCS stored in ``datamodel.meta.wcsinfo``
+  from datamodel's tweaked GWCS. [#6936, #6947, #6955]
```
Comment (zacharyburnett) [inline CHANGES.rst:80]:
```suggestion
  from ``datamodel``'s tweaked GWCS. [#6936, #6947, #6955]
```

File: CHANGES.rst (hunk 1 above)
Comment (hbushouse) [reply CHANGES.rst]: All suggestions implemented in latest commit.

**Pull request merged**